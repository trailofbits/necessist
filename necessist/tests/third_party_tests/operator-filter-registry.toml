url = "https://github.com/ProjectOpenSea/operator-filter-registry"
