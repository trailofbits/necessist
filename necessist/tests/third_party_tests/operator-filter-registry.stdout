312 candidates in 19 test files
